src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DATA_SEPARATOR = ",";
+    private static final int OPERATION_TYPE_COLUMN_INDEX = 0;
+    private static final int AMOUNT_COLUMN_INDEX = 1;
+    private static final String SUPPLY_STRING = "supply";
+    private static final String BUE_STRING = "buy";
+    private static final String RESULT_STRING = "result";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String EMPTY_STRING = "";
+    private StringBuilder stringBuilder = new StringBuilder();
+    private int supplyAmount = 0;
+    private int buyAmount = 0;
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    private void readDataFromFile(String fromFileName) {
```

Comment:
Remember  about the [ordering class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents) (private methods  after public)